syslog-ng git snapshot fails to compile on SLES 12

While trying to compile today's git snapshot on SLES 12, I ran into this:
```
[  107s] libtool: compile:  gcc -std=gnu11 -DHAVE_CONFIG_H -I. -I./lib -I./modules -I./lib -I./modules -I./modules/json -I./modules/json -pthread -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_32 -I./lib/eventlog/src -I./lib/eventlog/src -DOPENSSL_LOAD_CONF -D_BSD_SOURCE -D__BSD_SOURCE -D__FAVOR_BSD -DHAVE_NET_ETHERNET_H -D_DEFAULT_SOURCE -I/usr/include/dbi -I/usr/include/dbi/dbi -isystem ./lib/ivykis/src/include -isystem ./lib/ivykis/src/include -D_GNU_SOURCE -D_DEFAULT_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -Wshadow -Wno-stack-protector -Wno-unused-parameter -Wno-variadic-macros -I/usr/include/json-c -O2 -g -pthread -fmessage-length=0 -grecord-gcc-switches -fstack-protector -O2 -Wall -D_FORTIFY_SOURCE=2 -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -c modules/json/json-parser-grammar.c  -fPIC -DPIC -o modules/json/.libs/libjson_plugin_la-json-parser-grammar.o
[  107s] /bin/sh ./libtool  --tag=CC   --mode=compile gcc -std=gnu11 -DHAVE_CONFIG_H -I.  -I./lib -I./modules -I./lib -I./modules -I./modules/json -I./modules/json -pthread -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include  -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_32 -I./lib/eventlog/src -I./lib/eventlog/src  -DOPENSSL_LOAD_CONF  -D_BSD_SOURCE -D__BSD_SOURCE -D__FAVOR_BSD -DHAVE_NET_ETHERNET_H -D_DEFAULT_SOURCE -I/usr/include/dbi -I/usr/include/dbi/dbi  -isystem ./lib/ivykis/src/include -isystem ./lib/ivykis/src/include  -D_GNU_SOURCE -D_DEFAULT_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -Wshadow -Wno-stack-protector -Wno-unused-parameter -Wno-variadic-macros   -I/usr/include/json-c  -O2 -g -pthread -fmessage-length=0 -grecord-gcc-switches -fstack-protector -O2 -Wall -D_FORTIFY_SOURCE=2 -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -c -o modules/json/libjson_plugin_la-json-parser-parser.lo `test -f 'modules/json/json-parser-parser.c' || echo './'`modules/json/json-parser-parser.c
[  107s] modules/json/format-json.c: In function 'tf_json_append_value_base64_encode':
[  107s] modules/json/format-json.c:226:27: error: 'save' undeclared (first use in this function)
[  107s]    if (((unsigned char *) &save)[0] == 1)
[  107s]                            ^
[  107s] modules/json/format-json.c:226:27: note: each undeclared identifier is reported only once for each function it appears in
[  107s] Makefile:21297: recipe for target 'modules/json/libjson_plugin_la-format-json.lo' failed
[  107s] make[2]: *** [modules/json/libjson_plugin_la-format-json.lo] Error 1
[  107s] make[2]: *** Waiting for unfinished jobs....
[  107s] libtool: compile:  gcc -std=gnu11 -DHAVE_CONFIG_H -I. -I./lib -I./modules -I./lib -I./modules -I./modules/json -I./modules/json -pthread -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_32 -I./lib/eventlog/src -I./lib/eventlog/src -DOPENSSL_LOAD_CONF -D_BSD_SOURCE -D__BSD_SOURCE -D__FAVOR_BSD -DHAVE_NET_ETHERNET_H -D_DEFAULT_SOURCE -I/usr/include/dbi -I/usr/include/dbi/dbi -isystem ./lib/ivykis/src/include -isystem ./lib/ivykis/src/include -D_GNU_SOURCE -D_DEFAULT_SOURCE -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -Wshadow -Wno-stack-protector -Wno-unused-parameter -Wno-variadic-macros -I/usr/include/json-c -O2 -g -pthread -fmessage-length=0 -grecord-gcc-switches -fstack-protector -O2 -Wall -D_FORTIFY_SOURCE=2 -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -c modules/json/json-parser-parser.c  -fPIC -DPIC -o modules/json/.libs/libjson_plugin_la-json-parser-parser.o
[  107s] Makefile:26776: recipe for target 'all-recursive' failed
[  107s] make[1]: *** [all-recursive] Error 1
[  107s] Makefile:11492: recipe for target 'all' failed
[  107s] make: *** [all] Error 2
```

--- a/modules/json/format-json.c
+++ b/modules/json/format-json.c
@@ -223,8 +223,8 @@ tf_json_append_value_base64_encode(const gchar *name, const gchar *value, gsize 
 
 #if !GLIB_CHECK_VERSION(2, 54, 0)
   /* See modules/basicfuncs/str-funcs.c: tf_base64encode() */
-  if (((unsigned char *) &save)[0] == 1)
-    ((unsigned char *) &save)[2] = 0;
+  if (((unsigned char *) &encode_save)[0] == 1)
+    ((unsigned char *) &encode_save)[2] = 0;
 #endif
 
   out_len += g_base64_encode_close(FALSE, state->buffer->str + init_len + out_len, &encode_state, &encode_save);

